_posts/wookiemage/2016-2-3-Colins-Chapter-5.md

```diff
@@ -0,0 +1,13 @@
+---
+author: wookiemage
+layout: post
+title: "Colin's Chapter 5 Exercises
+"
+---
+  Here's the first exercise:
+  <iframe src="https://trinket.io/embed/python/39cbbd5989" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+  Here's the second:
+  <iframe src="https://trinket.io/embed/python/f584d23e33" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+  These weren't too tricky. The hardest thing I kept running into was indentation errors. I would get it working and then try to set it inside the loop, or inside the Try/Except, but I couldn't get the danged indentation to line up. I ended up manually setting everything at no indentation and then moving it in based on how deeply it was nested. It was tedious, but it solved my issues.
```

Comment:
Hey tedious gets the job done sometimes.

If you were using tab and shift tab and things didn't like up sometimes there's a space on the front of the line that needs to be deleted.  Shift tabbing everything away does this, but there is a less drastic solution too :)